autoload/sl.vim

```diff
@@ -79,7 +78,9 @@ function! sl#animate()
   let l:wrap_backup = &l:wrap
   setlocal nowrap
   redraw
-  call sound_playfile(filepath)
+  if exists('*sound_playfile')
+    let l:filepath = expand('**/sl.wav')
+    call sound_playfile(filepath)
```

Comment:
endif

Reply:
Sorry...

Reply:
No prob :smile:

Reply:
Thank you very much.
I fix some problem.
Please check this code.

Reply:
Thanks. So you should move up sl.wav from autoload/sl.wav to base directory?

Reply:
OK. I will transfer this wav file to base directory.

Reply:
I fixed. Please check!